raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx

```
@@ -120,37 +120,38 @@ const RdsProgressBar = (props: RdsProgressBarProps) => {
-                                className={`step ${index < completedSteps
-                                    ? 'bg-completed'
-                                    : index === completedSteps && props.colorVariant === 'error'
-                                    ? 'bg-error'
-                                    : index === completedSteps && props.colorVariant === 'primary'
-                                    ? 'bg-inprogress'
-                                    : 'bg-default'
-                                    }`}
-                                    id="dash"
+                                className={`step ${
+                                    index < completedSteps
+                                        ? props.colorVariant === 'error'
```

ESLint

[error] Strings must use doublequote.

(quotes)